app/services/katello/pulp/rpm.rb

```diff
@@ -30,7 +30,7 @@ def provides
       end
 
       def files
-        if pulp_facts['files']
+        if pulp_facts['files'] && pulp_facts['files'].present?
```

Comment:
oh i see,  {}.present? returns false.  would it make more sense to check that the 'file' and 'dir' keys are there?  It could be non-empty but contain 'dir' for example.